cfgov/unprocessed/apps/filing-instruction-guide/package.json

```diff
@@ -13,7 +13,7 @@
   "private": "true",
   "license": "SEE LICENSE IN TERMS.md",
   "dependencies": {
-    "ctrl-f": "1.3.2"
+    "ctrl-f": "1.4.2"
```

Comment:
What's up with the versioning history? Moot now, but curious why it's not `1.4.0`

<img width="295" alt="Screen Shot 2023-06-13 at 12 38 53 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/fee78059-24e5-4cc8-85f4-08abc1d64eee">

Reply:
That's because I was distracted on a call and typed `1.4.2` instead of `1.4.0` after running `yarn publish`. 😑